tensorflow/contrib/distribute/python/examples/simple_estimator_example.py

```diff
@@ -69,9 +69,10 @@ def main(_):
   # Since there are 2 devices and 10 samples, we set steps=5.
   steps = 5
 
-  def train_input_fn():
-    features = tf.data.Dataset.from_tensors([[1.]]).repeat(10)
-    labels = tf.data.Dataset.from_tensors([1.]).repeat(10)
+  def train_input_fn(input_context):
+    batch_size = input_context.get_per_replica_batch_size(2)
```

Comment:
I am not sure whether this change is necessary since it is very dummy. If you think so, would you mind adding more comments for what this line is doing?

Reply:
OK. It is just an example not the test. I will add some comments here to address this function.